README.md

```diff
@@ -180,6 +180,7 @@ Set up your GitHub Actions workflow with a specific version of your programming
 - [Apply templates with Jinja2](https://github.com/cuchi/jinja2-action) - Use the Jinja2 template engine to generate files from templates.
 - [Has Changes](https://github.com/UnicornGlobal/has-changes-action) - Check if there are code changes from previous steps.
 - [Mind Your Language Action](https://github.com/tailaiw/mind-your-language-action) - Detect offensive comments in issues and pull requests, and warn senders.
+- [Linguist](https://github.com/fabasoad/linguist-action) - Checks a repository and produces information about used languages in output.
```

Comment:
Looks good but you will have to fix merge conflicts for each action once it's been merged in to this list.